wye/profiles/models.py

```diff
@@ -91,9 +91,18 @@ def get_user_type(self):
         return [x.slug for x in self.usertype.all()]
 
     @property
+    def get_user_type_for_display(self):
+        return ', '.join(self.get_user_type)
+
+    @property
     def get_interested_locations(self):
         return [x.name for x in self.interested_locations.all()]
 
+    @property
+    def get_interested_locations_for_display(self):
+        return ', '.join(self.get_interested_locations)
```

Comment:
How about using `|join:", "` in the template itself? This will help keep the presentation logic separate from models.